packages/react-components/react-button/src/stories/SplitButton/SplitButtonSizeMedium.stories.tsx

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { MenuButtonProps, SplitButton } from '@fluentui/react-button';
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, Tooltip, SplitButton } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger, SplitButton, Tooltip } from '@fluentui/react-components';
```